code/sorting/selection_sort/SelectionSort.php

```diff
@@ -1,29 +1,34 @@
 <?php
 // Part of Cosmos by OpenGenus Foundation
-function selection_sort($arr) {
-    $n = count($arr);
-
-    foreach ($arr as $k => $el) {
-        $min = $k;
-
-        for ($j = $k + 1; $j < $n; $j++) {
-            if ($arr[$j] < $arr[$min]) {
-                $min = $j;
-            }
-        }
-
-        $hold = $el;
-        $el = $arr[$min];
-        $arr[$min] = $hold;
-    }
-
-    return $arr;
+//this does not work as written.
+//selction sort involves finding the min of an array and to keep removing until you have a sorted array
+//so I will rewrite this
+//first create a function which traverses the array and does the pair by pair switch
+//then selection_sort will call that function the length of the array.  Assumes a O(n^2) runtime
```

Comment:
These comments should be in your PR description rather than in the code file itself.

Reply:
Why?  This description of what the code does should be part of its comments,   Especially in a situation when describing what an algorithm or data structure does.

Reply:
I will be happy to add this the PR but the comments should stay.

Reply:
Done.

Reply:
The description of what the code does is fine, but "this does not work as written" and "so I will rewrite this" makes no sense if it gets merged. Also if you do wish to describe what your code does you'll probably need to be more descriptive, and put it a cleaner manner. I do not think code in Cosmos usually contains large descriptions with them.

Reply:
@parthamittra Pls submit after removing those comments.